01-Calcium_Imaging/02-Imported Tables - Interactive.ipynb

```diff
@@ -67,8 +66,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import os\n",
```

Comment:
Please `jupytext` sync this notebook with the paired Python script.  If `jupytext` is an added burden, we can remove the paired Python scripts.